app/assets/javascripts/application.js

```diff
@@ -148,10 +148,14 @@ function mark_params_override() {
   $('a[rel="popover"]').popover();
 }
 
-function add_fields(target, association, content) {
+function add_fields(target, association, content, direction = 'append') {
```

Comment:
I don't think we can utilize default attributes in legacy JS ( IE 10 will not support it ).
we would need the old way here:
 ```suggestion
function add_fields(target, association, content, direction) {
  direction = direction ? direction : 'append';
```
This works for not passing the last argument, because javascript does not care about such things like argument parity, it will just pass `undefined` if you forgot one or few arguments (or throw some away if you pass more then expected)

Reply:
yeah, good idea, accepting